manifest/sourcefile.py

```diff
@@ -344,6 +344,32 @@ def content_is_css_manual(self):
         # return True if the intersection between the two sets is non-empty
         return bool(self.css_flags & {"animated", "font", "history", "interact", "paged", "speech", "userstyle"})
 
+    @cached_property
+    def spec_link_nodes(self):
+        """List of ElementTree Elements corresponding to nodes representing a
+        <link rel=help>, used to point to specs"""
+        if not self.root:
+            return []
+        return self.root.findall(".//{http://www.w3.org/1999/xhtml}link[@rel='help']")
+
+    @cached_property
+    def spec_links(self):
+        """Set of spec links specified in the file"""
+        rv = set()
+        for item in self.spec_link_nodes:
+            if "href" in item.attrib:
+                rv.add(item.attrib["href"])
+        return rv
+
+    @cached_property
+    def content_is_css_visual(self):
```

Comment:
This doesn't work in the short-term when we want it to work on csswg-test. Or are we just not caring about that any more?